js/dungeon-raid-v2.js

```diff
@@ -173,17 +181,40 @@
 		 * Mouse up event handler
 		 */
 		mouseUpHandler: function () {
-			var opts = this.options;
+			var opts = this.options,
+                objs = this.objects;
 
 			if (opts.dragActive) {
 				opts.dragActive = false;
 				if (opts.activeTiles.length > 2) {
 					this.deleteActiveTiles();
 				}
 				this.resetActiveTiles();
+
+                objs.collectedBlock.classList.remove('visible');
 			}
 		},
 
+        /**
+         * Mouse move event handler
+         */
+        mouseMoveHandler:  function (e) {
+            var opts = this.options,
+                objs = this.objects;
+
+            if (opts.dragActive) {
+                e.preventDefault();
+                e.stopPropagation();
+
+                // console.log(e.changedTouches[0]);
+                if(e.changedTouches && e.changedTouches[0]){
+                    objs.collectedBlock.style.transform = 'translate('+ (15 + e.changedTouches[0].pageX) + 'px, ' + (15 + e.changedTouches[0].pageY) + 'px)';
```

Comment:
Please avoid any magic numbers like 15
Create a varible in game settings for this